blazor/sparkline/getting-started.md

```diff
@@ -17,42 +17,41 @@ This section briefly explains about how to include [Blazor Sparkline](https://ww
 
 ## Create a new Blazor App in Visual Studio
 
-You can create **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio in one of the following ways,
+You can create a **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio via [Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0) or the [Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-integration/template-studio).
 
-* [Create a Project using Microsoft Templates](https://docs.microsoft.com/en-us/aspnet/core/blazor/tooling?pivots=windows)
+## Install Syncfusion Blazor Sparkline and Theme NuGet in the App
```

Comment:
Do need Theme NuGet?

Reply:
Not need Theme NuGet also removed the Theme NuGet from this PR.